ElasticaAdmin/BlockParameter/ElasticSearchStrategy.php

```diff
@@ -17,7 +19,7 @@ class ElasticSearchStrategy implements BlockParameterInterface
      */
     public function support(BlockInterface $block)
     {
-        return in_array($block->getComponent(), array ('elastica_search', 'elastica_list'));
+        return in_array($block->getComponent(), array (self::NAME, 'elastica_list'));
```

Comment:
What about the 'elastica_list' string ? I think there must be somewhere a ElasticaListStrategy to update too